source/Octopus.Cli/Commands/ApiCommand.cs

```diff
@@ -303,13 +303,23 @@ protected List<string> ReadAdditionalInputsFromConfigurationFile(string configFi
 
         protected static IEnumerable<string> FormatReleasePropertiesAsStrings(ReleaseResource release)
         {
-            return new List<string>
+            var releaseProperties = new List<string>
             {
                 "Version: " + release.Version,
                 "Assembled: " + release.Assembled,
                 "Package Versions: " + GetPackageVersionsAsString(release.SelectedPackages),
                 "Release Notes: " + GetReleaseNotes(release)
             };
+            if (!string.IsNullOrEmpty(release.VersionControlReference?.GitRef))
+            {
+                releaseProperties.Add("Git Reference: " + release.VersionControlReference.GitRef);
+            }
+            if (!string.IsNullOrEmpty(release.VersionControlReference?.GitCommit))
+            {
+                releaseProperties.Add("Git Commit: " + release.VersionControlReference.GitCommit);
```

Comment:
hmm first impression was to have more details like - 
```
Git Commit (resolved from Git Reference): ... 
```

But I think for someone running this command, it's fairly self explanatory that that's what it is. 🤷‍♂️  so yeah not too fussed if it's just `Git Commit: ...`